server/routes/blog.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express'
+const router = express.Router()
+import blogController from '../controllers/blogController'
+import multer from 'multer'
+const storage = multer.diskStorage({
```

Comment:
Great work @muhireheir! Anyway, as a feedback; I hope these codes can be well maintained if you create a helper that uploads an image. What do you think?